### DISCUSSION TITLE
Close scene with Windows + W

### DISCUSSION THREAD

**Discussion by pc9098** [godotengine/godot-proposals#10717] [Editor]
In many web browsers like Chrome (or even softwares like Aseprite) you can close windows with Windows + W.
Would be nice to be implemented
![image](https://github.com/user-attachments/assets/4a11bb49-e595-4a4c-b250-861de33bbe1f)

**Comment by Mickeon** [comment]
You may be mistaking it for `CTRL + W`. `Windows + W` does not close a Chrome window.
In Godot, by default, this shortcut exists and is bound to `Ctrl + Shift + W`, because `Ctrl + W` closes a script in the Script Editor.

**Reply by Calinou** [reply]
>  In Godot, by default, this shortcut exists and is bound to Ctrl + Shift + W, because Ctrl + W closes a script in the Script Editor.

We could make <kbd>Ctrl + W</kbd> close the current scene if not in the Script main screen, while keeping <kbd>Ctrl + Shift + W</kbd> functional regardless of the current main screen.
